test/integration/webhook/v1alpha2/workload_test.go

```diff
@@ -125,7 +126,12 @@ var _ = ginkgo.Describe("Workload validating webhook", func() {
 			ginkgo.By("Updating the priority")
 			priority := int32(10)
 			workload.Spec.Priority = &priority
-			gomega.Expect(k8sClient.Update(ctx, workload)).Should(gomega.Succeed())
+			gomega.Eventually(func() error {
+				var newWL kueue.Workload
+				gomega.Expect(k8sClient.Get(ctx, client.ObjectKeyFromObject(workload), &newWL)).To(gomega.Succeed())
+				newWL.Spec.Priority = &priority
```

Comment:
```suggestion
				newWL.Spec.Priority = pointer.Int32(10)
```

Also remove the priority.